ci_handbook/model-output-standards.qmd

```diff
@@ -0,0 +1,45 @@
+---
+title: "{{< fa arrow-right >}} Model Output Standards"
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    toc-position: left
+reading-time: true
+image: "../images/Lego_dimensions.svg.png"
```

Comment:
I think this is the wrong image, you want the honeycomb here. I'm seeing the lego pieces which should be under 'modularity'

Reply:
/images/hive.jpg

Reply:
I do want the lego pieces as the precise dimensions of the lego pieces are required to build something.